### DISCUSSION TITLE
Post about breaking changes or obsolete.

### DISCUSSION THREAD

**Discussion by xavierjohn** [vkhorikov/CSharpFunctionalExtensions#456] [Q&A]
It would be nice to see an announcement on why something is being replaced with something else so that we can update the code accordingly.
When we update the nuget package we run into issues like 
Replace OnFailure() with TapError() and I have no idea why and what does it mean to update.

Thanks.

**Comment by vkhorikov** [comment]
This particular one is just a rename, to match the D#-esque convention. We never implementation (at least that's the goal), in order to maintain backward compatibility.

**Reply by xavierjohn** [reply]
I am not familiar with D#-esque.  
Do you mean C# style?

**Reply by vkhorikov** [reply]
It's a typo, I meant F#. This library uses naming/terms from F#.

**Reply by vkhorikov** [reply]
In fact, from F# and TypeScript, because there are some functions in TS that don't exist in F#, as far as I know.
